src/app.js

```diff
@@ -226,6 +227,31 @@ class App {
     });
   }
 
+  loadAll(objs) {
+    let modelObjs = [];
+    modelObjs = [...objs];
+    this.showSpinner();
+    this.dropEl.replaceChildren();
+
+    Promise.all(
+      modelObjs.map((fileObj) => {
+        return fetch(fileObj.link)
+          .then((res) => res.blob())
+          .then((blob) => {
+            return new File([blob], fileObj.filename, { type: "" });
+          });
+      })
+    ).then((fileList) => {
+      const length = fileList.length;
+      const fileMap = new Map(fileList.map((file) => [file.name, file]));
+      this.load(fileMap);
+      setInterval(() => {
+        const randomIndex = Math.floor(Math.random() * length);
+        this.loadCurrentModel(randomIndex);
```

Comment:
This loads a random model, but it can also repeat the same model multiple times.

What we want is for a model to be shuffled in a random order so that each model is viewed once each loop.

Reply:
@jjangsangy I updated the code to avoid duplicate

Reply:
So this isn't what I meant.

Let's say you have 5 models at index `[0, 1, 2, 3, 4, 5]`

We want to shuffle the index `[5, 3, 4, 1, 2]`. We want to shuffle the order and keep that order.